packages/kbn-reporting/export_types_helpers_public/build_kibana_path.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+interface Args {
+  basePath: string;
+  appPath: string;
+  spaceId?: string;
+}
+
+export const buildKibanaPath = ({ basePath, appPath, spaceId }: Args) => {
```

Comment:
duplicated file - also exists in `packages/kbn-reporting/common/build_kibana_path.ts`